pkg/ledger/ledger.go

```diff
@@ -26,17 +26,35 @@ var DefaultContracts = []core.Contract{
 }
 
 type Ledger struct {
-	locker  Locker
-	store   storage.Store
-	monitor Monitor
+	locker              Locker
+	store               storage.Store
+	monitor             Monitor
+	allowPastTimestamps bool
 }
 
-func NewLedger(store storage.Store, locker Locker, monitor Monitor) (*Ledger, error) {
-	return &Ledger{
+type LedgerOption = func(*Ledger)
+
+func WithPastTimestamps(l *Ledger) {
+	l.allowPastTimestamps = true
+}
+
+func NewLedger(
```

Comment:
no tests to update here ?

Reply:
New argument in last position so keeping the default elsewhere, but just added a new test in process_test.go to better cover the usage of those options